src/Authentication/Provider/JwtAuth.php

```diff
@@ -62,8 +62,8 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager, Jwt
    * {@inheritdoc}
    */
 	public function applies(Request $request) {
-    $auth = $request->headers->get('Authorization');
-		return preg_match('/^Bearer .+/', $auth);
+      $auth = $request->headers->get('Authorization');
+      return preg_match('/^Bearer .+/', $auth);
```

Comment:
the formatting in that file is a little f*#$(*& anyway. Its using some mix of tabs and spaces.

That line was my attempt at a quick fix. I'll rebase this pull and take that out, then submit another with all the tabs removed in this file.